java-connectors/kafka-connect-common/src/main/java/io/lenses/streamreactor/common/util/InputStreamHandler.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017-2024 Lenses.io Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.lenses.streamreactor.common.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Helper class to allow easy manipulation of InputStreams.
+ */
+public class InputStreamHandler {
+
+  /**
+   * Extracts String from InputStream byte buffer.
+   *
+   * @param inputStream {@link InputStream to read}
+   * @return String representation of inputStream
+   */
+  public static String extractString(InputStream inputStream) {
+    int bufferSize = 1024;
```

Comment:
Why 1024?

Reply:
I just wanted it to be not small enough (this way we'll have to read multiple times cause the buffer won't be sufficiently large) and not too large (1024 `char`s is not too much even if you'd ever want to run it on a low-memory unit).

Nothing more really :)

Reply:
The reason for the question is to prompt you to add a comment in the code giving context as to why this value was chosen.